gems/pending/util/vcenter_thumb_print.rb

```diff
@@ -0,0 +1,25 @@
+require 'util/thumb_print'
+
+class VcenterThumbPrint < ThumbPrint
```

Comment:
If `VMwareWebService` is a no-go because of the cross-lib dependencies it'd introduce, I guess we should add a generic `gems/pending/vmware` directory for now. `util` currently seems to be *almost* clean of individual-provider stuff, so it's probably better that we avoid adding more -- they'll just interfere with future gemification efforts.